.idea/codeStyleSettings.xml

```diff
@@ -135,9 +138,8 @@
           <option name="PARENT_SETTINGS_INSTALLED" value="true" />
           <indentOptions>
             <option name="INDENT_SIZE" value="2" />
-            <option name="CONTINUATION_INDENT_SIZE" value="2" />
+            <option name="CONTINUATION_INDENT_SIZE" value="4" />
```

Comment:
I used that setting at "2" before, but it resulted in large reformatting of the file. So I altered the setting so that it matches the file's layout (which was a result of @dkincade initial formatting - I assume he's never wrong :) ).